packages/calypso-e2e/src/lib/test-account.ts

```diff
@@ -3,26 +3,29 @@ import path from 'path';
 import chalk from 'chalk';
 import { BrowserContext, Page } from 'playwright';
 import { TestAccountName } from '..';
-import { getAccountCredential, getAccountSiteURL, getCalypsoURL } from '../data-helper';
+import { getAccountSiteURL, getCalypsoURL } from '../data-helper';
+import { EmailClient } from '../email-client';
 import envVariables from '../env-variables';
+import { SecretsManager } from '../secrets';
 import { TOTPClient } from '../totp-client';
 import { LoginPage } from './pages/login-page';
-import type { AccountCredentials } from '../types/data-helper.types';
+// import type { AccountCredentials } from '../types/data-helper.types';
```

Comment:
```suggestion
```

Reply:
Ah my bad, thanks for catching that.

Reply:
Addressed in 2f93fdf836bdf423be0769629391e14c14208962.